File: spec/code_samples/Python_python3/alerts@{alert_type}/delete.py
import http.client

conn = http.client.HTTPSConnection("www.directfreight.com")

headers = {
    'accept': "application/json",
    'content-type': "application/json",
    'end-user-token': "SOME_STRING_VALUE"
    }

conn.request("DELETE", "/api/alerts/%7Balert_type%7D?alert_id=SOME_ARRAY_VALUE", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: spec/code_samples/Php_http1/alerts@{alert_type}/delete.php
<?php

$request = new HttpRequest();
$request->setUrl('https://www.directfreight.com/api/alerts/%7Balert_type%7D');
$request->setMethod(HTTP_METH_DELETE);

$request->setQueryData(array(
  'alert_id' => 'SOME_ARRAY_VALUE'
));

$request->setHeaders(array(
  'end-user-token' => 'SOME_STRING_VALUE'
));

try {
  $response = $request->send();

  echo $response->getBody();
} catch (HttpException $ex) {
  echo $ex;
}